app/app.rb

```diff
@@ -0,0 +1,86 @@
+ENV["RACK_ENV"] ||= "development"
+
+require 'sinatra/base'
+require 'sinatra/flash'
+
+require_relative 'datamapper_setup'
+
+class Chitter < Sinatra::Base
```

Comment:
Now that we've moved onto more complex database apps, it feels like the cleanliness of my code and separation of concerns has got a bit lost. Is this controller okay in terms of design and slimness? What would be the best and highest priority ways of refactoring my controller here?